TalonNav-LPS.md

```diff
@@ -15,43 +15,53 @@ DCPU-16 Hardware Info:
 
 
 Description:
-    LPS is an entirely local positioning system, using a beacon and receiver
-    setup. This device will lock onto the nearest beacon with a specified ID,
-    using a doppler reciever to triangulate both position and velocity relative
-    to the beacon.
+    The Talon Navigation Systems LPS is an entirely local positioning system, using 
+    a beacon and receiver model. This device uses a doppler receiver to triangulate
+    position and velocity relative a specified beacon.
 
-    The Talon LPS has a massive range of 16km, and is accurate to the nearest mm.
+    The Talon has a range of 16km, and is accurate to 1 mm.
     Guaranteed to lock on in under 5 seconds or your money back!
+
 (Note: Guarantee applies only in free space with fully functioning Talon Navigation
-Systems LPS Beacon within 16km of receiver, presence of multiple beacons with same ID
+Systems LPS Beacon within 16km of receiver. Presence of multiple beacons with same ID
 may lead to undefined behaivour and voids warranty.)
 
 Interrupt behavior:
-    When a HWI is received by the LPS, it reads the A register and does one
+    When a HWI is received by the Talon, it reads the A register and does one
```

Comment:
Just referring to this device as "the Talon" doesn't contextually really make much sense, since Talon is the company that makes the LPS, PPS and other Beacons.